X1Plus/X1Plus
# kexec_ui OTA autoinstall

This branch adds logic and dialog to kexec_ui and /kexec/check_kexec to make it possible to "auto install" an x1p file. Once you have set the OTA flag in settings.json, the next time you reboot your device it will start installation automatically.

How to test:
1) Install the artifact build from this PR
2) Find an x1p file you want to install and save the filename to `ota.filename`. You can do this via ssh by running the following command, replacing '1.1rc1.x1p' with your target x1p filename.
```
python3 /opt/x1plus-test-settings.py ota.filename=1.1rc1.x1p
```

3) Reboot or power cycle and there should be no further input necessary to complete the update.

Any failures the updater experiences still require manual reboots, but any potential issues with wifi can be avoided by ensuring that the base firmware the x1p requires is on the sd card.
